src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/Nodes/NodeFilter.cs

```diff
@@ -76,6 +78,22 @@ public string ToPropertyFilterString()
                     filter = $"{filter} {orPrefix} @{Property} = {proeprtyValue}";
                 }
             }
+            filter = $"({filter})";
+
+            return filter;
+        }
+
+        public static string ConcatProperties(IEnumerable<NodeFilter> filters)
+        {
+            string filter = "";
+            var list = filters.ToList();
+            for (int i = 0; i < list.Count; i++)
+            {
+                var value = list[i];
+                
+                string orPrefix = i == 0 ? "" : "and";
```

Comment:
should this be named andPrefix?  (also, as a minor nit, I typically use string.Empty instead of "" literal for readability).